### ISSUE TITLE
make install fails following Quickstart for Go-based Operators

### ISSUE THREAD

**Issue by adambkaplan** [operator-framework/operator-sdk#4453]
When following the quickstart guide using operator-sdk v1.3 (modified for the operator I'm trying to build), I get the following error from `make install`:

```
        * accumulateFile error: "accumulating resources from 'bases/operator.shipwright.io_shipwrightbuilds.yaml': evalsymlink failure on '/var/home/adkaplan/go/src/github.com/shipwright-io/operator/config/crd/bases/operator.shipwright.io_shipwrightbuilds.yaml' : lstat /var/home/adkaplan/go/src/github.com/shipwright-io/operator/config/crd/bases/operator.shipwright.io_shipwrightbuilds.yaml: no such file or directory"
        * loader.New error: "error loading bases/operator.shipwright.io_shipwrightbuilds.yaml with git: url lacks host: bases/operator.shipwright.io_shipwrightbuilds.yaml, dir: evalsymlink failure on '/var/home/adkaplan/go/src/github.com/shipwright-io/operator/config/crd/bases/operator.shipwright.io_shipwrightbuilds.yaml' : lstat /var/home/adkaplan/go/src/github.com/shipwright-io/operator/config/crd/bases/operator.shipwright.io_shipwrightbuilds.yaml: no such file or directory, get: invalid source string: bases/operator.shipwright.io_shipwrightbuilds.yaml"
```

I see a file in confg/crc/bases called `operator.shipwright-io_shipwrightbuildren.yaml` instead.

**Steps to reproduce**

1. Init repo with operator-sdk:

```
$ operator-sdk init --domain=shipwright.io --repo=github.com/shipwright-io/operator
```

2. Create an API

```
$ operator-sdk create api --group operator --version v1alpha1 --kind ShipwrightBuild --resource=true --controller=true
```

3. Create the image and push via `make docker-build` and `make docker-push`
4. Run `make install`

**Comment by jmrodri** [comment]
@adambkaplan I was able to recreate the problem. It seems to happen when `make manifests` is run. The `operator.shipwright-io_shipwrightbuildren.yaml` file is being created instead of `operator.shipwright-io_shipwrightbuilds.yaml` as if something was doing bad pluralization.

The workaround I was able to do was:
1. Rename the file `mv operator.shipwright.io_shipwrightbuildren.yaml operator.shipwright.io_shipwrightbuilds.yaml`
2. Edi the file to reference builds: `vi config/crd/bases/operator.shipwright.io_shipwrightbuilds.yaml ` change all references to `buildren` to `builds`.
3. Run `make install`

**Comment by jmrodri** [comment]
I created a repo with 3 commits showing the 3 steps I've taken to recreate the issue:
https://github.com/jmrodri/shipwright/commit/149fafbc2bc2871f9519da16bd5aa88d659a0462

The third commit: `make manifests` is causing the problem.

**Comment by jmrodri** [comment]
We're using `controller-gen` v0.4.1 which depends on `gobuffalo/flect` v0.2.0 for pluralization logic. `gobuffalo/flect` v0.2.0 has a problem:

```
[jesusr@transam flecttest]$ ./flecttest 
builds
shipwrightbuildren
```
With `gobuffalo/flect` v0.2.2, it is fixed.

```
[jesusr@transam flecttest]$ ./flecttest 
builds
shipwrightbuilds
```

**Comment by jmrodri** [comment]
Looks like the fix was done after the v0.4.1 release https://github.com/kubernetes-sigs/controller-tools/pull/522

**Comment by camilamacedo86** [comment]
It will be fixed when we update/align SDK with upstream: WIP https://github.com/operator-framework/operator-sdk/pull/4402

**Comment by estroz** [comment]
@adambkaplan what plugin version are you using (the `layout` field in your PROJECT file)? #4402 won't actually fix this unless you are using the proper version of `controller-gen` (v0.4.1) which is only supported by the go/v3 plugin (introduced in operator-sdk v1.3.0). v0.4.0+ creatse v1 CRDs and webhooks by default, so you can use v0.4.1 if you add `crdVersions={v1beta1}` to your Makefile and `webhookVersions={v1beta1}` to your `api/<version>/<kind>_webhook.go` markers. Really https://github.com/kubernetes-sigs/controller-tools/pull/522 should be backported to v0.3.0.

@camilamacedo86 can you remove the `Closes #4453` from #4402?

Since this is fixed upstream I'm going to close this issue.

**Comment by estroz** [comment]
Actually re-opening this to make sure this is fixed, since you mentioned operator-sdk v1.3 was used.
